Server content/Resources/Prototypes/Roles/Jobs/Command/blue_shield.yml

```
@@ -0,0 +1,43 @@
+- type: job
+  id: Blueshield
+  name: job-name-blueshield  
+  description: job-description-blueshield  
+  playTimeTracker: JobBlueshield  
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Engineering
+      time: 54000 # Corvax-RoleTime
+    - !type:DepartmentTimeRequirement
+      department: Medical
+      time: 54000 # Corvax-RoleTime
+    - !type:DepartmentTimeRequirement
+      department: Security
+      time: 54000 # Corvax-RoleTime
+    - !type:OverallPlaytimeRequirement
+      time: 324000 # Corvax-RoleTime
```

```suggestion
      time: 648000 # Corvax-RoleTime
```
Это ОЧЕНЬ важная должность